WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -16293,10 +16291,8 @@
 			files = (
 			);
 			inputFileListPaths = (
-				"$(SRCROOT)/Credentials/secrets-manifest.xcfilelist",
 			);
 			inputPaths = (
-				"$(SRCROOT)/../Scripts/BuildPhases/ApplyConfiguration.sh",
```

Comment:
Interestingly, I had to keep the output paths parameter, otherwise the build would fail, but only when run through `xcodebuild`, not from Xcode.

![image](https://user-images.githubusercontent.com/1218433/124550597-3c0d2880-de74-11eb-8df4-6f6cab8c7589.png)

```
❌  error: Build input file cannot be found: '/Users/gio/Developer/a8c/wpios/DerivedData/Build/Intermediates.noindex/ArchiveIntermediates/Jetpack/BuildProductsPath/Secrets/Secrets.swift' (in target 'Jetpack' from project 'WordPress')
```

Reply:
Yep, that's how outputPaths works and what it's for.

I think the fact that it didn't fail from Xcode GUI is a bug/glitch, maybe because the project was already build from your `xcodebuild` test and didn't have to incremental-build when you tried in Xcode?

But the goal is to tell Xcode which files this script phase generates, so that it can resolve the dependency graph when figuring out which targets, dependencies and phases need to be run and which can be skipped in incremental builds if their output are already up-to-date.

It also allows Xcode to know that this file – which is one of the input of the "Compile Sources" build phase, so which needs to be present before that phase runs – will be generated by this script. Since Xcode does the full dependency graph analysis of the various build phases first, and only runs them once it has resolved the whole graph (both in order to avoid starting to build stuff only to fail later if it can predict earlier on that the failure will happen, and to optimize build time in incremental builds by determining what can be skipped), you need to fill in that Output Files here to let it resolve the dependency graph and optimize incremental builds properly.